static/src/javascripts/lib/fetch-json.ts

```diff
@@ -0,0 +1,40 @@
+import config_ from 'lib/config';
+import fetch from 'lib/fetch';
+
+// This is really a hacky workaround ⚠️
+const config = config_ as {
+	get: (s: string, d?: string) => string;
+};
+
+const fetchJson = async (
+	resource: string,
+	init: RequestInit = {},
+): Promise<unknown> => {
+	if (typeof resource !== 'string')
+		throw new Error('First argument should be of type `string`');
+
+	let path = '';
+	if (!RegExp('^(https?:)?//').exec(resource)) {
+		path = String(config.get('page.ajaxUrl', '')) + resource;
+		init.mode = 'cors';
+		init.credentials = 'include';
+	}
+	const resp = (await fetch(path, init)) as Response;
```

Comment:
If `resource` _does_ match the regex, `path` will never be assigned a new value, so the request will always be made with an empty string. This is breaking all uses of `fetchJson` that pass a full URL as `resource`.

Reply:
You’re right, I’ve raised a fix in #23693.